connect-api/src/main/java/com/ifttt/connect/api/ErrorResponse.java

```diff
@@ -19,4 +19,9 @@ public ErrorResponse(String code, String message) {
         this.code = code;
         this.message = message;
     }
+
+    @Override
+    public String toString() {
+        return "ErrorResponse{" + "code='" + code + '\'' + ", message='" + message + '\'' + '}';
+    }
```

Comment:
can remove this now.

Reply:
Might as well keep it here for the future. Having readable toString isn't that bad.